mern/server/DAO/QA_dao.js

```diff
@@ -0,0 +1,60 @@
+import QA from "../models/question_answer.js";
+
+//Class of DAO functions related to the QA entities
+export default class QA_DAO {
+  //Empty constructor
+  constructor(props) {
```

Comment:
You don't need a super here since there is no inheritance. Props in also unnecessary since there is an empty constructor.,

Reply:
Nice. Good to know